client/lib/signup/test/flow-controller.js

```diff
@@ -168,20 +168,6 @@ describe( 'flow-controller', () => {
 				store.dispatch( submitSignupStep( { stepName: 'userCreation' } ) );
 			} );
 		} );
-
-		test.skip( 'should throw an error when the flow is completed without all dependencies provided', () => {
```

Comment:
Disabled in #32982

Reply:
I don't remember the details from more than a year ago, but the tests probably depended on some Flux implementation details that proved very difficult to rewrite in Redux. :+1: